docs/hydra/index.md

```diff
@@ -16,160 +16,126 @@ are.
+
+With OAuth, clients can request access to resources on a server, and the owner
+of these resources can grant the requested access together with dedicated
+credentials. In our example, you could grant the printing service read-only
+access to your photos (only your photos, not your friend list) on the social
+network. These credentials come in the form of an access token -- a string
+denoting a specific scope, lifetime, and other access attributes. The client
+(printing service) can use this access token to request the protected resources
+(your photos) from the resource server (the social network).
 
 ### What is OpenID Connect 1.0?
 
-OpenID Connect 1.0 is a simple identity layer on top of the OAuth 2.0 protocol.
-It enables Clients to verify the identity of the End-User based on the
-authentication performed by an Authorization Server, as well as to obtain basic
-profile information about the End-User in an interoperable and REST-like manner.
-
-As background, the OAuth 2.0 Authorization Framework and OAuth 2.0 Bearer Token
-Usage specifications provide a general framework for third-party applications to
-obtain and use limited access to HTTP resources. They define mechanisms to
-obtain and use Access Tokens to access resources but do not define standard
-methods to provide identity information. Notably, without profiling OAuth 2.0,
-it is incapable of providing information about the authentication of an
-End-User.
-
-OpenID Connect implements authentication as an extension to the OAuth 2.0
-authorization process.
-
-Source
-[OpenID Connect Core 1.0](http://openid.net/specs/openid-connect-core-1_0.html)
-
-OpenID Connect allows clients of all types, including Web-based, mobile, and
-JavaScript clients, to request and receive information about authenticated
-sessions and end-users. The specification suite is extensible, allowing
-participants to use optional features such as encryption of identity data,
-discovery of OpenID Providers, and session management, when it makes sense for
-them.
-
-There are different work flows for OpenID Connect 1.0, we recommend checking out
+OAuth 2.0 is a complex protocol for authorizing access to resources. If all you
+need is authentication, OpenID Connect 1.0 enables clients to verify the
+identity of the end user based on the authentication performed by an
+Authorization Server and obtain basic profile information in an interoperable
+and REST-like manner.
+
+OpenID Connect allows clients of all types, including web and mobile, to receive
+information about authenticated sessions and end users. The specification is
+extensible, allowing participants to add encryption of identity data, discovery
+of OpenID Providers, and session management as needed.
+
+There are different work flows for OpenID Connect 1.0. We recommend checking out
 the OpenID Connect sandbox at [openidconnect.net](https://openidconnect.net/).
 
+A more detailed introduction of both OAuth 2.0 and OpenID Connect is available
```

Comment:
The links to external videos are now down here, below the (shortened) summaries of OAuth2 and OpenID Connect.